sdk/maps/Azure.Maps.Search/CHANGELOG.md

```diff
@@ -2,13 +2,13 @@
 
 ## 2.0.0-beta.5 (Unreleased)
 
-### Features Added
-
 ### Breaking Changes
 
+- Support Search API `2025-01-01`. Support `StreetName` and `StreetNumber` in `Address` class and remove unused types
```

Comment:
API churn between betas is not classified as a breaking change, and we discourage listing them in the change log as such.   We want to be sure that any rare scenario that results in a true breaking change to stable versions is accentuated and not harder to find due to these sections for beta churn.